netcode.c

```diff
@@ -4055,10 +4162,19 @@ void netcode_server_connect_client( struct netcode_server_t * server,
 
     char address_string[NETCODE_MAX_ADDRESS_STRING_LENGTH];
 
-    netcode_printf( NETCODE_LOG_LEVEL_INFO, "server accepted client %s %.16" PRIx64 " in slot %d\n", 
-        netcode_address_to_string( address, address_string ), client_id, client_index );
+    /* TODO: try to find a way to deliver match itd. */
+    if ( !skillz_add_client_to_match( server, 111, client_id ) )
```

Comment:
The matchId should be present within the connect token at some point.

Reply:
Yeah saving that for later.

Reply:
Otherwise 🌮 a lot of context switching for me, so I could have missed things :(